### PR TITLE
[4.0] Workflow transitions layout 2

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
This PR resolves conflicts from  https://github.com/joomla/joomla-cms/pull/22315.
Please close the former PR.

### CODE REVIEW

File: administrator/components/com_workflow/tmpl/transitions/default.php [hunk 1]
```diff
@@ -126,18 +127,50 @@
...
-									<?php if ($item->from_stage_id < 0) : ?>
-										<?php echo Text::_('JALL'); ?>
-									<?php else : ?>
-										<?php echo $this->escape(Text::_($item->from_stage)); ?>
-									<?php endif; ?>
+									<td class="nowrap">
+										<?php if ($item->from_stage_id < 0) : ?>
+											<?php echo Text::_('JALL'); ?>
+										<?php else : ?>
+											<?php
```
Comment (SharkyKZ) [inline administrator/components/com_workflow/tmpl/transitions/default.php]: Remove trailing space.

File: administrator/components/com_workflow/tmpl/transitions/default.php [hunk 2]
```diff
@@ -126,18 +127,50 @@
...
+											endif; ?>
+											<span class="<?php echo $icon; ?>" aria-hidden="true"></span>
+											<span class="sr-only"><?php echo TEXT::_('COM_WORKFLOW_CONDITION') . $condition; ?></span>
+											<?php echo ' ' . $this->escape(Text::_($item->from_stage)); ?>
+										<?php endif; ?>
 									</td>
-									<td class="text-center">
-										<?php echo $this->escape(Text::_($item->to_stage)); ?>
+									<td class="nowrap">
+										<?php
```
Comment (SharkyKZ) [inline administrator/components/com_workflow/tmpl/transitions/default.php]: Remove trailing space.

File: administrator/components/com_workflow/tmpl/transitions/default.php [hunk 3]
```diff
@@ -126,18 +127,50 @@
...
-										<?php echo $this->escape(Text::_($item->from_stage)); ?>
-									<?php endif; ?>
+									<td class="nowrap">
+										<?php if ($item->from_stage_id < 0) : ?>
+											<?php echo Text::_('JALL'); ?>
+										<?php else : ?>
+											<?php 
+											if ($item->from_condition == Workflow::CONDITION_ARCHIVED):
+												$icon = 'icon-archive';
+												$condition = TEXT::_('JARCHIVED');
```
Comment (SharkyKZ) [inline administrator/components/com_workflow/tmpl/transitions/default.php]: Change all instances of `TEXT` to `Text`.

File: administrator/components/com_workflow/tmpl/transitions/default.php (hunk 2 above)
Comment (chmst) [reply administrator/components/com_workflow/tmpl/transitions/default.php]: sry, I don't see here a trailing space?

Comment (SharkyKZ) [reply administrator/components/com_workflow/tmpl/transitions/default.php]: After `<?php`.

Comment (chmst) [reply administrator/components/com_workflow/tmpl/transitions/default.php]: thank you!

**Comment by lavipr** [comment]
I have tested this item :white_check_mark: successfully on 27ccd1687d1048d6f991009ec0f56c03d8fe92e4<br /><br />I compared the view transition list before and after the patch. Icons are there after the patch.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22343">issues.joomla.org/tracker/joomla-cms/22343</a>.</sub>

**Comment by bembelimen** [comment]
I have tested this item :white_check_mark: successfully on 27ccd1687d1048d6f991009ec0f56c03d8fe92e4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22343">issues.joomla.org/tracker/joomla-cms/22343</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by wilsonge** [comment]
Thanks. Apologies for making you do the remake!

**Pull request merged**